README.rst

```diff
@@ -1,76 +1,61 @@
-=======
-Astropy
-=======
-
-.. container::
+|Astropy Logo|
 
-    |Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
+|User Stats|
 
-The Astropy Project (https://astropy.org/) is a community effort to develop a
-single core package for Astronomy in Python and foster interoperability between
-Python astronomy packages. This repository contains the core package which is
-intended to contain much of the core functionality and some common tools needed
-for performing astronomy and astrophysics with Python.
+----
 
-Table of Contents
-=================
+|Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
+----
+The Astropy Project is a community effort to develop a
+single core package for astronomy in Python and foster interoperability between packages used in the field.
+This repository contains the core library.
 
-- `Installation <#installation>`_
-- `Contributing <#contributing>`_
-- `Getting Started with GitHub Codespaces <#getting-started-with-github-codespaces>`_
-- `Supporting the Project <#supporting-the-project>`_
-- `License <#license>`_
+* `Website <https://astropy.org/>`_
+* `Documentation <https://docs.astropy.org/>`_
+* `Slack <https://astropy.slack.com/>`_
+* `Open Astronomy Discourse <https://community.openastronomy.org/c/astropy/8>`_
+* `Astropy users mailing list <http://mail.python.org/mailman/listinfo/astropy>`_
+* `Astropy developers mailing list <https://groups.google.com/g/astropy-dev>`_
+* `Astropy community engagement study <https://zenodo.org/records/10603049>`_
+* `Astropy diversity, equity and inclusion study <https://astropy-dei.orgmycology.com/>`_
```

Comment:
I was thinking about adding these under `impact_health.rst`, not necessarily in README like this, but I wonder what others think.

Reply:
Ah sorry I thought you had meant the README. Though yeah I think they're nice here actually; the `impact_health` page is really just about codebase development.

Reply:
I am still not convinced but I will defer to other reviewers.

Reply:
No strong opinions on this -- I think it's fine to have in the README, but also makes sense to be in `impact_health.rst`.

Reply:
Sounds good, I've moved the links to these out of the README and into `impact_health.rst`, with some text around them, in [bfd95a7](https://github.com/astropy/astropy/pull/16822/commits/bfd95a71b7e6c7926e4e4e8578ae36259ecb3e7d).